packages/core/src/app/payment/createPaymentFormService.ts

```diff
@@ -9,11 +9,20 @@ import { FormContextType } from '@bigcommerce/checkout/ui';
 import { PaymentContextProps } from './PaymentContext';
 
 export default function createPaymentFormService(
-    formikContext: FormikContext<PaymentFormValues>,
+    formikContext: FormikContextType<PaymentFormValues>,
     formContext: FormContextType,
     paymentContext: PaymentContextProps,
 ): PaymentFormService {
-    const { setFieldTouched, setFieldValue, submitForm, validateForm, values } = formikContext;
+    const {
+        setFieldTouched: formikSetFieldTouched,
+        setFieldValue: formikSetFieldValue,
+        submitForm,
+        validateForm,
+        values,
+    } = formikContext;
+
+    const setFieldTouched = formikSetFieldTouched as PaymentFormService['setFieldTouched'];
```

Comment:
Why do we need to map `setFieldTouched` to `formikSetFieldTouched` and then do this here?

Reply:
It's ok, just wanted to check why. I am ok with this but maybe we move it to L40 so it becomes less confusing?